### DISCUSSION TITLE
希望能有安卓版

### DISCUSSION THREAD

**Discussion by shuntou** [xfangfang/wiliwili#91] [Ideas]
希望能有安卓版

**Comment by xfangfang** [comment]
@taotaolion 相关问题已经讨论过很多次了：https://github.com/xfangfang/wiliwili/issues?q=android

推荐使用：https://github.com/xiaye13579/BBLL

---

我已经在issue区内评论过很多次了，任何安卓版的支持都会导致wiliwili的其他任何全平台版一起被封禁，作者会被连累律师函，已经有相当多的实例案件。

wiliwili从来都是丰富官方客户端不支持的平台而不是在某些平台替代官方客户端，尤其是安卓端的替代，会导致官方损失大量利益，被封禁是必然。

在这里我再次声明，我从未以任何方式发布过wiliwili的安卓版，并且只要有任何wiliwili的安卓版出现在互联网，我都会立即关闭wiliwili的github仓库不再维护。

**Reply by shuntou** [reply]
谢谢

**Reply by tansichong** [reply]
你好，请问有开发xbox版本的计划吗
